src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageControl.xaml.cs

```diff
@@ -48,21 +48,20 @@ private void OnClearEnvironmentVariableGridError(object sender, EventArgs e)
             ClearGridError(dataGridEnvironmentVariables);
         }
 
-        private void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
+        private async void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
```

Comment:
Wow. I didn't realize BeginInvoke delegates will crash the process. Interesting.